.github/actions/build/action.yml

```diff
@@ -50,6 +58,12 @@ runs:
     - if: ${{ env.CODECOV }} == "on"
       name: Upload report to Codecov
       uses: codecov/codecov-action@v3
+    - name: Test Leiningen plugin
+      shell: bash
+      working-directory: ${{ github.workspace }}/lein-jank
+      run: |
+        ${{ github.workspace }}/compiler+runtime/bin/install
+        ./test.sh
```

Comment:
Please move the script to `lein-jank/bin/ci/test` (no `.sh`) to fit our existing structure and naming.